alignak/daemons/brokerdaemon.py

```diff
@@ -449,8 +453,10 @@ def setup_new_conf(self):  # pylint: disable=R0915,R0912
 
         :return: None
         """
+
         with self.conf_lock:
             conf = unserialize(self.new_conf, True)
+            logger.warning("Configuration: %s", conf)
```

Comment:
I do agree, it may be important and not as useful as expected. I remove this log :/